src/stores/game.js

```diff
@@ -505,6 +506,11 @@ export const useGameStore = defineStore('game', {
         this.updateGame(res.body.game);
         return handleInGameEvents(res.body, route);
       } catch (err) {
+        // Swallow 401 error so we can authenticate from GameView
+        if (authStore.mustReauthenticate) {
+          this.id = gameId;
+          return;
+        }
```

Comment:
I tried to refactor `requestGameState()` to work the same way but it failed some of the reconnecting tests. I wrote up #1229 to make that func consistent w/ this one so we can get this improvement up and running